### DISCUSSION TITLE
My Github education student developer pack has been approved for days now but havent got the benefits offers till now no mail of benefits from github even

### DISCUSSION THREAD

**Discussion by GOAT17A** [community/community#138830] [GitHub Education]
### Hi everyone,

My Github education student developer pack has been approved for days now but havent got the benefits offers till now no mail of benefits from github even

**Comment by davevad93** [comment]
Hi @GOAT17A ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @GOAT17A was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @GOAT17A was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

**Reply by bnistor4edu** [reply]
I just want to congrats you for your patience in responding 10000 times the same thing to people. ❤️

**Comment by barishaxxer** [comment]
Github has stated  `We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.` Probably you r experiencing this because new semester starts.Send them a mail as the article mentions.
https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved
